.env

```diff
@@ -0,0 +1,3 @@
+POSTGRES_PASSWORD=mawuli302
```

Comment:
Add this file to the .gitIgnore. We did this to hide these credentials.
Add a .env.example instead

Reply:
done